cmake.deps/CMakeLists.txt

```
@@ -146,6 +140,20 @@ endif()
 include(ExternalProject)
 set_directory_properties(PROPERTIES EP_PREFIX "${DEPS_BUILD_DIR}")
 
+set(GLOBAL_CMAKE_ARGS
```

if these are only used by deps, let's use the `DEPS_` prefix, which already has precedent.
```suggestion
set(DEPS_CMAKE_ARGS
```

---

Good call.